### ISSUE TITLE
GDPR's right to information must include context

### ISSUE THREAD

**Issue by tsteur** [piwik/piwik#20685]
## Summary

see https://noyb.eu/en/noyb-win-cjeu-right-access-may-include-documents-and-database-extracts

> It is also important that the CJEU emphasizes the need to contextualize the information provided. It is not enough to for companies to send a mere list of raw data. The data must be provided to the data subject in full, in context, and true to the original.

However, our GDPR export currently only provides raw data.

To better provide context, we would ideally refer to https://developer.matomo.org/guides/database-schema that explains all the raw data fields and tables.

Can we embed a link to the DB schema so data subjects (end users) are able to get some context? 

Since this is legally relevant, it be great to work on this soon.

## Possible solutions

We could maybe mention this here: 
<img width="580" alt="image" src="https://user-images.githubusercontent.com/273120/236706710-9280a6f4-8521-44a8-ad8d-fe9a85a0690f.png">
and/or maybe add a sentence referring to the DB schema to the export after the JSON output?

**Comment by michalkleiner** [comment]
Hi @tsteur,
assigning this enhancement to the team for prioritisation and decision on where to add the link or how to tweak the content.

**Comment by Stan-vw** [comment]
Just to check if I understand your proposed solution correctly, your proposal is to:
1. Add a sentence referring to the DB schema (location: add it to the export, after the JSON output)
2. Add a link to the DB schema (so data subjects (end users) are able to get some context, location: GDPR tools page)

My initial thought is to simplify it to:
1. Add a sentence referring to the DB schema (location: GDPR tools page)
2. This sentence has a link to the DB schema (so data subjects (end users) are able to get some context)
3. Draft sentence: To fully understand the data and its context, you can [download our database schema](https://LinkToDBSchema) ([learn more](https://LinkToNewFAQ)). 
4. Add an FAQ that explains the schema in an easy to digest manner (potentially keeping it high level and referring to developer documentation for more specifics)

**Comment by tsteur** [comment]
@Stan-vw yes those were some thoughts. Both would have a link to the DB schema. I didn't think about it extensively though and just a random thought. There may be better ways.

Draft sentence for the page: We could make people aware that for GDPR compliance they should provide data subjects ideally with context and they can do this using the DB schema link or so.

**Comment by Stan-vw** [comment]
ah yes, it's not about the matomo user getting the DB extract, now I see your reason for adding it to the downloaded JSON.
How about:
1. Add a sentence referring to the new FAQ (location: GDPR tools page)
2. Draft sentence: The GDPR dictates the need to contextualize the information provided to data subjects. It is not enough to for companies to send a mere list of raw data. The data must be provided to the data subject in full, in context, and true to the original ([learn more](https://linktonewfaq/)).
3. Add a sentence referring to the new FAQ (location: JSON download)
4. Draft sentence: To understand this data, please refer to the Matomo database schema ([learn more](https://linktonewfaq/)).
5. Create a new FAQ that explains why this FAQ url is added (to provide context) and explains the schema in an easy to digest manner (potentially keeping it high level and referring to developer documentation for more specifics), and perhaps a link to download the schema

**Comment by tsteur** [comment]
Sounds great 👍

**Comment by randy-innocraft** [comment]
Hi @tsteur. Thank you for creating the issue and bringing this to our attention, that's very appreciated. We have reviewed and triaged this ticket internally. Our team will prioritise this, and we will update you on the progress here when we have an update to share. If you have any further information or questions, please feel free to add them here.
